bin/compile_model.sh

```diff
@@ -174,6 +180,8 @@ elif [ "$SITE" == "hexagon" ] ; then
    if [[ -z "${ESMF_DIR}" ]] ; then
       # use as default
        export ESMF_DIR=/homeappl/home/pr2n0112/HYCOM_TOOLS/esmf/ESMF.6.3.0rp1
+      #export ESMF_DIR=/home/nersc/knutali/opt/esmf_5_2_0rp3-nonetcdf/
+      #export ESMF_DIR=/home/nersc/knutali/opt/esmf_6_3_0rp1/
```

Comment:
Here and some other places: we can just remove this old paths and have a cleaner code.

Reply:
I agree that whether it can be done by the reviewer?

Reply:
Jiping, you need to do the changes and commit them, then I can review it again.